api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -717,3 +728,14 @@ message PerRouteConfig {
     string requirement_name = 2 [(validate.rules).string = {min_len: 1}];
   }
 }
+
+// This message specifies a combination of header name and claim name.
+message JwtClaimToHeader {
+  // The HTTP header name to copy the claim to.
+  string header_name = 1
+      [(validate.rules).string = {min_len: 1 well_known_regex: HTTP_HEADER_NAME strict: false}];
+
+  // The field name for the JWT Claim : it can be a nested claim of type (eg. "claim.nested.key", "sub")
+  // String separated with "." in case of nested claims or a single value.
```

Comment:
remove "or a single value"
maybe:
The nested claim name must use dot "." to separate the JSON name path.